### ISSUE TITLE
Set alarm planned skill doesn't work after google speech to text

### ISSUE THREAD

**Issue by alok760** [fossasia/susi_server#1361]
**Actual Behaviour**
The example of `set alarm in one minute` planned skill works for exactly the same words used, but testing in the smart speaker results in changing the query slightly resulting in the skill not working.

**Expected Behaviour**
Google STT changes the query `set alarm in one minute` to `set alarm in 1 minute`. - note the numeric one
Planned skill should work for `set alarm in 1 minute`

**Steps to reproduce it**
- https://api.susi.ai/susi/chat.json?timezoneOffset=-330&q=set+an+alarm+in+one+minute -  *WORKS*
- https://api.susi.ai/susi/chat.json?timezoneOffset=-330&q=set+an+alarm+in+1+minute - *DOESN'T WORK*

**Comment by fragm3** [comment]
No, you can pick it up. Skills are stored under repository [susi_skill_data](https://github.com/fossasia/susi_skill_data)

**Comment by vaibhavmatta** [comment]
I am claiming this issue now!

**Comment by alok760** [comment]
@vaibhavmatta here's the concerned file - https://github.com/fossasia/susi_server/blob/development/conf/os_skills/test/en/alarm.txt

You can make this skill more intuitive or just add `set alarm in 1 minute` for now as a temporary test solution
